packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1166,13 +1196,25 @@ class RawEditorState extends EditorState
     }
     // Snapshot the input before using `await`.
     // See https://github.com/flutter/flutter/issues/11427
-    final ClipboardData? data = await Clipboard.getData(Clipboard.kTextPlain);
-    if (data == null) {
+    final data = await widget.clipboardManager.getData();
+    if (data?.plainText == null && data?.delta == null) {
       return;
     }
 
-    _replaceText(
-        ReplaceTextIntent(textEditingValue, data.text!, selection, cause));
+    if (data!.delta != null) {
```

Comment:
may add `hasDelta` to `FleatherClipboardData`? (`hasHtml` also?)

Reply:
Check https://github.com/fleather-editor/fleather/pull/233#discussion_r1464335956 for 'hasDelta'.